generator/post_generation_scripts/go.sh

```diff
@@ -1,3 +1,4 @@
 #!/bin/sh
 sed -i 's/go 1.13/go 1.18/g' "$(find out/qovery-client-go/ -type f)"
 cd out/qovery-client-go || exit ; go get ; go mod tidy ; gofmt -s -w .
+go build
```

Comment:
It's to avoid the issue you spoke about this morning ?

Reply:
Yes, his will make sure that the generated code can be compiled. 
This will avoid pushing code that doesn't compile to the go client repo 👌